packages/types/src/workflow/common.ts

```diff
@@ -0,0 +1,10 @@
+export interface WorkflowInputConfig {
```

Comment:
comment: Think we should reconsider renaming to be more descriptive, but not important for this PR

Reply:
Definitely 💯